nakama/nakama.lua

```diff
@@ -235,19 +235,40 @@ function M.create_user_group_list_user_group(
 	}
 end
 
+--------------------------------------------------------------------------------
+--- create_validated_purchase_environment
+-- - UNKNOWN: Unknown environment.  - SANDBOX: Sandbox/test environment.  - PRODUCTION: Production environment.
+function M.create_validated_purchase_environment(
```

Comment:
Should this be empty?

Reply:
No, probably not :-) It's generated code and comes from this spec:

https://github.com/heroiclabs/nakama/blob/master/apigrpc/apigrpc.swagger.json#L3281-L3291

I see that the `enum` property is new and only used here and in the other two places where there's a empty functions. I'll take a look at the code generator and make sure `enum` is handled.

Reply:
@britzl yep take a look at this helper: https://github.com/heroiclabs/nakama-dotnet/blob/master/codegen/main.go#L632

Reply:
I'm now detecting enums and handling these appropriately